compredict/client.py

```diff
@@ -80,8 +81,24 @@ def verify_peer(self, option: str):
     def last_error(self) -> Error:
         return self.connection.last_error
 
+    def set_callback_urls(self, callback_url: Union[list, str]) -> str:
+        """
+        Accept list of urls and format them into one string with dividing '|' in between.
+        This is the format accepted by ai core.
+        :param callback_url: list of callback urls
+        :return: one callback_url string
+        """
+
+        if type(callback_url) == list:
```

Comment:
Here it is better to use isinstance(callback_url, list) than type(callback_url). Here is [why](https://switowski.com/blog/type-vs-isinstance) FYI.

Reply:
Cool. Didn't know about it. Done.